examples/volumes/glusterfs/README.md

```diff
@@ -84,17 +87,13 @@ You can verify that the pod is running:
 $ kubectl get pods
 NAME             READY     STATUS    RESTARTS   AGE
 glusterfs        1/1       Running   0          3m
-
-$ kubectl get pods glusterfs -t '{{.status.hostIP}}{{"\n"}}'
-10.240.169.172
 ```
 
-You may ssh to the host (the hostIP) and run 'mount' to see if the Glusterfs volume is mounted,
+You may execute the command `mount` inside the container to see if the GlusterFS volume is mounted correctly:
```

Comment:
How come? I believe `kubectl exec <pod> -- <cmd>` runs `<cmd>` inside the container. See also [Running Commands in a Container with kubectl exec](https://kubernetes.io/docs/tasks/kubectl/get-shell-running-container/).

For instance when I execute this command, I also see the processes from within the container:

```
$ kubectl exec hello-2096222913-x8p6b -- ps
PID   USER     TIME   COMMAND
    1 root       0:00 /usr/bin/hello
   19 root       0:00 ps
```